src/qat/purr/backends/qblox/device.py

```diff
@@ -120,12 +129,12 @@ def __init__(
         dev_id: str = None,
         name: str = None,
         address: str = None,
-        cfg: Dict = None,
+        dummy_cfg: Dict = None,
     ):
         super().__init__(id_=dev_id or os.environ.get("QBLOX_DEV_ID"))
         self.name = name or os.environ.get("QBLOX_DEV_NAME")
         self.address = address or os.environ.get("QBLOX_DEV_IP")
-        self.cfg = cfg
+        self.dummy_cfg = dummy_cfg
```

Comment:
Why `dummy`?

Reply:
config for dummy qblox cluster, it's a "mock" cluster qblox provides for testing